api.swagger.yaml

```diff
@@ -401,6 +396,43 @@ paths:
                     items:
                       $ref: '#/components/schemas/user-search-schema'
     description: Search for users. Returns user snippets. If there are no result, an empty array is returned.
+  /users/mention:
+    get:
+      tags:
+        - users
+      summary: Autocomplete usernames for @-mentioning.
+      description: Autocomplete usernames for @-mentioning. The text parameter matches against the start of the username. Full user objects are returned. The list may optionally be filtered by group to show only members of the group, to allow completing only users that can be mentioned when composing a group post. You can not filter by members of a private or secret group unless you are a member yourself. Webmasters can always be mentioned, so they are always matched even if they are not members of the group being filtered. Redacted users are never matched.
```

Comment:
I added a separate summary and description as the description was a bit long winded; I know we're not really using summary anywhere else but I think we probably should...